tools/vagrant/vagrant-libvirt.md

```diff
@@ -6,10 +6,11 @@ order: 2
 
 # Vagrant with libvirt support installation
 
-To use Vagrant with libvirt, you need to install the `vagrant-libvirt` package:
+To use Vagrant with libvirt, you need to install the `vagrant-libvirt` package and the `virtualization` group:
 
 ```
 $ sudo dnf install vagrant-libvirt
+$ sudo dnf group install virtualization
```

Comment:
I do not think this is needed. vagrant-libvirt should pull in all that is required.